modules/i3s/src/lib/parsers/parse-slpk/slpk-archieve.ts

```diff
@@ -37,6 +37,10 @@ const PATH_DESCRIPTIONS: {test: RegExp; extensions: string[]}[] = [
     test: /statistics\/f_\d+\/\d+$/,
     extensions: ['.json.gz']
   },
+  {
+    test: /statistics\/summary$/,
```

Comment:
This path is not a part of the spec https://docs.ogc.org/cs/17-014r9/17-014r9.html#_example_1_2_slpk_structure_summary_for_3d_objects

Reply:
Oh, I didn't know that but I see it now. I create this pull request based on fix @dariaterekhova-actionengine. I described the issu in https://github.com/visgl/loaders.gl/discussions/3148#discussioncomment-11121415.

Reply:
I don't understand that slpk archive from [Esri/i3s-speci](https://github.com/Esri/i3s-spec/blob/master/docs/1.8/BSL_ReadMe.md) is not compatible with the standard. 
Am I missing something?

Reply:
If this is simply a clean extension of the standard (i.e. an additional optional path that provides additional, non-critical information), then there is no compatibility issues. 

It would however be nice to separate these things in the code (using some lightweight mechanism separate files, functions, tables etc) and call it out in comments.

It would also be nice to cover it in the documentation as well:
- Add a section that lists Esri additions to the standard
- Mention which ones are supported, and how (if this path is present, how do we use it and what extra capabilities are offered to the user as a result).